source/Camera/Basic/connectCamera.m

```diff
@@ -0,0 +1,46 @@
+function camera = connectCamera(application,folder)
+
+    % Camera = connectCamera(application)
+    %
+    % Connect to Zivid Camera
+    %
+    % INPUT:
+    % application - Zivid Application
+    % folder - path to Zivid\bin folder, default: C:\Program Files\Zivid\bin
+    %
+    % OUTPUT:
+    % camera - Zivid Camera
+
+    global ZIVID_CAMERA
+
+    if isempty(ZIVID_CAMERA)
+
+        if nargin < 1
+            application = [];
+        end
+
+        if nargin < 2
+            if isempty(which('ZividNET.dll'))
+                folder = 'C:\Program Files\Zivid\bin';
+            else
+                folder = which('ZividNET.dll');
+                folder = folder(1:end-13);
+            end
+        end
+
+        if isempty(application)
+            application = ZividNETinitialize(folder);
+        end
+
+        current = cd;
+
+        cd(folder)
+        goback = onCleanup(@() cd(current));
+
+        ZIVID_CAMERA = application.ConnectCamera;
+
+    end
+
+    camera = ZIVID_CAMERA;
+
+end
```

Comment:
I'm not so sure I would have had this as it save the camera globally. So f.ex multizivid is not possible. I like using:
app = ZividApplication();
cam = app.ConnectCamera();
Matlab will get problems if you clear the workspace without calling cam.Dispose(or Disconnect?) in this case.

Reply:
Do you suggest then to completely get rid of connectCamera.m function?
What about the zividApplication.m function? Would you not have the application global?

Reply:
You only need one application, so keeping that global is fine. The camera on the other hand might change.

F.ex, what happends if you run 

for i = 1: 2 %run twice
    app = zividApplication;
    cam = connectCam(app);
    cam.Disconnect;
end
??

Reply:
connectCamera() is removed. Instead, we will use camera = app.ConnectCamera;
In the tutorial, we will add a note on not to clear the memory before disconnecting.